homeassistant/components/orangepi_gpio/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "orangepi_gpio",
   "name": "Orange Pi GPIO",
   "documentation": "https://www.home-assistant.io/integrations/orangepi_gpio",
-  "requirements": ["OPi.GPIO==0.4.0"],
+  "requirements": ["OPi.GPIO==0.5.1"],
```

Comment:
When bumping a library add a link to the changelog, or at minimum a diff between library versions is added to the PR description.

Reply:
waiting for other PR, after that I will include changelog